lib/builderator/interface/berkshelf.rb

```diff
@@ -33,6 +33,10 @@ def berkshelf_config
       def source
         directory.join('Berksfile')
       end
+
+      def solver
```

Comment:
You shouldn't need this for the template. `<%= berkshelf.solver %>` actually reference the Config object directly now.